index.html

```diff
@@ -4,42 +4,38 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>spots</title>
-    <link type="text/css" rel="stylesheet" href="./pages/index.css" />
+<<<<<<< HEAD
+<<<<<<< HEAD
+    <link rel="icon" href="favicon.ico" type="image/x-icon" />
+=======
+    <link type="text/css" rel="stylesheet" href="./src/pages/index.css" />
+>>>>>>> parent of 3494d28 (fixed errors)
+=======
+>>>>>>> 3494d28 (fixed errors)
```

Comment:
* The app won't start because of merge conflicts https://snipboard.io/fre8an.jpg